docs/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Ipheco Chris Blog</title>
+    </head>
+    <body>
+        <header>
+            <h1>Ipheco Chris Blog</h1>
+            <nav class="menu">
+                <ul>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">About Me</a></li>
+                    <li><a href="#">Contact Us</a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <main>
```

Comment:
Not required!

Reply:
@promise-J
@ChidiPatrick
@OgJaykay
@Pieta9221
@MiniHack
@Melumm 
I have made the Changes to my Lesson 1 & 2 Task 1, Please Kindly Check it out.